src/protocol/bayang_nrf24l01.c

```diff
@@ -508,9 +534,9 @@ uintptr_t Bayang_Cmds(enum ProtoCmds cmd)
         initialize();
         return 0;
     case PROTOCMD_NUMCHAN:
-        return 12;
+        return (analogaux ? 14 : 12);
```

Comment:
This should always be set to 14

Reply:
Thanks. I should have looked closer to figure out if that should change or not... PROTOCMD_DEFAULT_NUMCHAN is okay to change with the feature, though?

Reply:
I think it will always return 12 because when creating is a new model analogaux == 0
Maybe better set it to 14 too ... and I4ll have to update the documentation

Reply:
Okay. Will returning 14 on both of these cause any problems for when analog aux is not enabled (Referring to two channels that don't exist)?

Reply:
if analogaux is disabled then they will never be read, and they would just return 0 anyway.